microsoft-365/security/defender-endpoint/machine-groups.md

```diff
@@ -73,7 +73,7 @@ As part of the process of creating a device group, you'll:
 
 ## Manage device groups
 
-You can promote or demote the rank of a device group so that it's given higher or lower priority during matching. When a device is matched to more than one group, it's added only to the highest ranked group. You can also edit and delete groups.
+You can promote or demote the rank of a device group so that it's given higher or lower priority during matching. A device group with a rank of 1 is the highest rank. When a device is matched to more than one group, it's added only to the highest ranked group. You can also edit and delete groups.
```

Comment:
```suggestion
You can promote or demote the rank of a device group so that it's given higher or lower priority during matching. A device group with a rank of 1 is the highest ranked group. When a device is matched to more than one group, it's added only to the highest ranked group. You can also edit and delete groups.
```